EIPS/eip-4671.md

```diff
@@ -0,0 +1,468 @@
+---
+eip: 4671
+title: Non-Tradable Tokens
+description: A standard interface for non-tradable tokens.
+author: Omar Aflak (@omaraflak), Pol-Malo Le Bris, Marvin Martin (@MarvinMartin24)
+discussions-to: https://ethereum-magicians.org/t/eip-4671-non-tradable-token/7976?u=omaraflak
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-01-13
+requires: 165
+---
+
+# Non-Tradable Token Standard
+
+<!-- AUTO-GENERATED-CONTENT:START (TOC) -->
+- [Simple Summary](#simple-summary)
+- [Abstract](#abstract)
+- [Motivation](#motivation)
+- [Specification](#specification)
+  - [Extensions](#extensions)
+    - [Metadata](#metadata)
+    - [Delegation](#delegation)
+- [Rationale](#rationale)
+  - [On-chain vs Off-chain](#on-chain-vs-off-chain)
+- [Implementation](#implementation)
+  - [NTT](#ntt)
+  - [NTTDelegate](#nttdelegate)
+- [NTT for EIP ?](#ntt-for-eip-)
+- [Copyright](#copyright)
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Simple Summary
+
+A standard interface for <u>**non-tradable tokens**</u>, aka <u>**NTT**</u>s.
+
```

Comment:
```suggestion
```
The `Simple Summary` section has been removed (in favor of the `description` section in the header)